ghga_datasteward_kit/s3_upload/file_decryption.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2021 - 2023 Universität Tübingen, DKFZ, EMBL, and Universität zu Köln
```

Comment:
Were the file decrypter, encrypte, and uploader rewritten? Or is GitHub just not able to detect that they where moved into different files? I.e. do I have to do a thorough review of them?

Reply:
It's a git issue. From everything that's moved into the s3_upload path, only config and entrypoint should have real changes.

Reply:
OK, thanks for the clarification.